docs/wip/theme-shadowing.md

```diff
@@ -1,7 +1,66 @@
-## Why it is important?
+# Shadowing
+This documentation will address Shadowing in Gatsby Themes focusing on how we use it in the Store Framework Jamstack (SFJ). You can find general explanations and examples of the Shadowing concept in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
```

Comment:
[Shadowing](https://www.gatsbyjs.com/docs/conceptual/how-shadowing-works/) is a Gatsby concept that allows high levels of rendering customization by letting users override components, objects, or any other element in a theme’s `src` directory.

>ℹ️ We strongly encourage you to check Gatsby's documentation and explore some [Shadowing](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/) examples.